router_macro_derive/src/modules.rs

```diff
@@ -3,7 +3,7 @@ use proc_macro_error::{abort, Diagnostic, Level};
 use crate::builder::get_string_from_attribute;
 use syn::{Attribute, Ident};
 
-pub fn modules_path(_: Ident, attrs: std::slice::Iter<'_, Attribute>) -> Option<String> {
+pub fn path(_: Ident, attrs: std::slice::Iter<'_, Attribute>) -> Option<String> {
```

Comment:
`modules::modules_path` is module name repetition

Reply:
yes you are so right, Thank you for this correction.